website/snippets/_microsoft-adapters-soon.md

```diff
@@ -0,0 +1,3 @@
+:::tip Coming soon
+dbt Cloud support for the Microsoft Fabric and Microsoft Synapse adapters is coming soon! 
```

Comment:
"coming soon" felt redundant given it's on the header for callout. also played w/ the language to make it more direct and clear? take it or leave it
```suggestion
Support for the Microsoft Fabric and Azure Synapse in dbt Cloud!
```

<img width="937" alt="image" src="https://github.com/dbt-labs/docs.getdbt.com/assets/8158673/518e0c49-991e-4db9-b22b-1c21fa88120b">